test/app/teams_test.rb

```diff
@@ -91,6 +91,17 @@ def test_user_must_be_manager
     end
   end
 
+  def test_logged_in_user_can_search_teams
+    [
+      [:get, '/teams'],
+      [:get, '/teams/']
+    ].each do |verb, endpoint|
```

Comment:
Opinions vary, but I really don't like looping in tests, it obscures what is being tested and doubles up on testing the "unimportant" bits.
Here you are testing:
1) that teams route works.
2) that the teams route works with a trailing slash.
3) that the response body contains an expected string (twice)
4) that the response status is 200 (twice)